resources/lang/nld.ini

```diff
@@ -1,46 +1,111 @@
-; Updated time : 26th 07 2017
+; Updated time : 7th 11 2019
 language.name = "Dutch"
 ; general
 divider = "========================================"
 spacer = " -=+=- {%0} -=+=- "
-error = "Er is iets fout gegaan"
-noperm = "U heeft geen rechten om deze operatie uit te voeren"
-runingame = "Voer deze operatie alstublieft in-game uit"
 commands = "Operaties"
-; user interfaces
-ui.confirmation = "Confirmatie"
-ui.confirmation.yes = "Ja"
-ui.confirmation.no = "Nee"
-; ui brush
+enabled = "ingeschakeld"
+disabled = "uitgeschakeld"
+confirmation = "Confirmatie"
+confirmation.yes = "Ja"
+confirmation.no = "Nee"
+; errors
+error = "Er is iets fout gegaan"
+error.command-error = "Het lijkt erop dat je een argument mist of de opdracht verkeerd hebt gebruikt!"
+error.runingame = "Voer deze operatie alstublieft in-game uit"
+error.limitexceeded = "U probeert te veel blokken tegelijk te bewerken. Verklein de selectie of verhoog de limiet"
+error.notarget = "Geen doelblok gevonden. Vergroot gereedschapsbereik met //setrange indien nodig"
+error.noselection = "Geen selectie gevonden - selecteer eerst een gebied"
+error.selectioninvalid = "De selectie is niet geldig! Controleer of alle pisities zijn geselecteerd!"
+error.nosession = "Er is geen sessie gemaakt - waarschijnlijk geen toestemming om {% 0} te gebruiken"
+error.noclipboard = "Geen klembord gevonden - maak eerst een klembord"
+warning.differentlevel = "[WAARSCHUWING] Je bewerkt een niveau waar je momenteel niet in zit!"
+; commands
+command.info.title = "Informatie"
+command.limit.current = "Huidig limiet: {%0}"
+command.limit.set = "Limiet voor wijziging van blok is ingesteld op {%0}"
```

Comment:
Might be better translated as 'Limiet voor aantal wijzigingen van blokken is ingesteld op {%0}'.